PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc

```diff
@@ -115,28 +115,28 @@ EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
 CAPTION "Search results"
 FONT 8, TEXT("MS Shell Dlg"), 0, 0, 0x0
 BEGIN
-DEFPUSHBUTTON   ">",IDC_INCFINDNXTOK,243,0,16,14, NOT WS_VISIBLE
+//DEFPUSHBUTTON   ">",IDC_INCFINDNXTOK,243,0,16,14, NOT WS_VISIBLE

```

Comment:
@ozone10,

Thank you for the "Dialogs Improvements" series. Great work! 👍 

It seems that commenting this line creates an issue:
_Find in Current Document_.
Again, _Find in Current Document_.

_Result_:
The entire text in the _Search Results_ window is selected.

Reply:
Good catch, I've added comment